### PR TITLE
Update Brazilian Portuguese Localization

### PR DESCRIPTION
**Pull request by Pixelnando**
Updated with the new difficulty strings (These were based on the official PT-BR translation of Doom Eternal).

### CODE REVIEW

File: Localization/pt-BR.hjson [hunk 1]
```diff
@@ -1,3 +1,15 @@
+UI: {
+	Journey: "Jornada"
+	Normal: "Muito Jovem Para Minerar"
+	Expert: "Um Ossinho Não Dói"
+	Master: "Eu Sou A Encarnação de Cthulhu"
```
Comment (Mirsario) [inline Localization/pt-BR.hjson:5]: The names of the difficulties seem to be translated well, but I've been questioning for a while now whether or not they should be translated at all.. What do you think? I tried translating them to my first language, and either couldn't apply the puns (Expert), couldn't keep obviousness of the references (Master), or couldn't make the punified translation rhyme / be nice to pronounce (Normal.)

(Sorry for the late reply)

Comment (Pixelnando) [reply Localization/pt-BR.hjson:5]: In Brazilian Portuguese, I feel like they fit pretty well. Have you checked if any versions of Doom (aka what these are referencing) have these translated to your first language? You could then base the translation around that.

File: Localization/pt-BR.hjson [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+
+	JourneyDescription: "O modo Criativo / Acessível (Para aqueles que querem estar no controle)"
+	NormalDescription: "Menos danos de inimigos, mesmas recompensas (Para os descompromissados)"
+	ExpertDescription: "A nova experiência padrão (Para aqueles que gostam de um desafio justo)"
+	MasterDescription: "Punição extrema e novas recompensas (Para os verdadeiramente desumanos)"
```
Comment (Mirsario) [inline Localization/pt-BR.hjson]: The keys must've changed in recent vanilla updates.. Or I'm dumb.
```suggestion
	WorldDescriptionJourney: "O modo Criativo / Acessível (Para aqueles que querem estar no controle)"
	WorldDescriptionNormal: "Menos danos de inimigos, mesmas recompensas (Para os descompromissados)"
	WorldDescriptionExpert: "A nova experiência padrão (Para aqueles que gostam de um desafio justo)"
	WorldDescriptionMaster: "Punição extrema e novas recompensas (Para os verdadeiramente desumanos)"
```

File: Localization/pt-BR.hjson (hunk 1 above)
Comment (Mirsario) [reply Localization/pt-BR.hjson:5]: ..After quite some thinking, the best localization I came up with for `Hurt/Bone Me Plenty` in Russian (original translated as approximately `Make Me Feel Pain` in Eternal), is pretty much just `Burn Me Alive`, and it's not even a pun on a classic game reference, but more so just a standalone reference to Overhaul, dang it.. Anyway, merging this. Thanks for the PR.

**Pull request merged**